### ISSUE TITLE
Define objects in io-package.json

### ISSUE THREAD

**Issue by GermanBluefox** [TA2k/ioBroker.nightscout#4]
It is better of all instance objects will be defined in io-package.json
https://github.com/TA2k/ioBroker.nightscout/blob/master/main.js#L50

like here: https://github.com/ioBroker/ioBroker.iot/blob/master/io-package.json#L189

**Comment by TA2k** [comment]
Yes but this objects id depends on the website your entered in the options. There are not really instance objects. You can enter later multiple website this means each website is one device.
